File: 1. Two Sum/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        l=[]
        i=0
        while(i!=len(nums)):
            x=target-nums[i]
            if(x not in nums):
                i+=1
            else:
                if(nums.index(x)!=i):
                    return([i,(nums.index(x))])
                else:
                    i+=1
            

File: 1.Two Sum/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) 
    {
        vector<int>ans;        
        unordered_map<int, int> index;
        int tar;
        
        for(int i=0;i<nums.size();i++)
        {   
            int t = target - nums[i];
            if (index.find(t) != index.end())
            {
                return {index[t], i};
            }
            
            index[nums[i]] = i;
        }
        
        
        return {};
        
    }
};
